tools/netsh/dllmain.c

```diff
@@ -77,11 +77,12 @@ CMD_ENTRY g_EbpfShowCommandTable[] = {
 #else
 typedef struct _CMD_ENTRY_ORIGINAL
 {
-    LPCWSTR pwszCmdToken;         // The token for the command
+    _Null_terminated_ const wchar_t* pwszCmdToken; // The token for the command
```

Comment:
I'm guessing Null_terminated is needed to prevent compiler warnings/errors?  Normally this would be annotated `_Field_z_` instead of `_Null_terminated_`.  See https://learn.microsoft.com/en-us/cpp/code-quality/annotating-structs-and-classes?view=msvc-170